language/move-ir/types/src/location.rs

```diff
@@ -78,7 +85,7 @@ impl<T> Spanned<T> {
     pub fn unsafe_no_loc(value: T) -> Spanned<T> {
         Spanned {
             value,
-            loc: Loc::new(Self::NO_LOC_FILE, Span::default()),
+            loc: Loc::new(Self::NO_LOC_FILE, 0, 0),
```

Comment:
Unrelated to this pull request, but I wonder what your current thoughts are on the concept of builtins and and other constructs having "no location?" Have you find this difficult to special-case in other parts of the project, or has it been fine?

Reply:
Not sure I quite understand the question, but this function is never really used for the `move-lang` crate, as you always need a loc to point to. For builtins or other operations, you just point to the current use case